kolibri/plugins/coach/assets/src/views/common/QuizLessonDetailsHeader.vue

```diff
@@ -27,6 +27,11 @@
       </div>
     </div>
     <MissingResourceAlert v-if="resource.missing_resource" />
+    <UiAlert v-if="isFromOldKolibiri">
```

Comment:
We should probably add `type="warning"` here - thoughts @radinamatic ?

The default "info" type results in a blue section w/ an info `(i)` icon, but `warning` is yellow w/ the `/!\` triangle exclamation icon.

Reply:
[Uploading pre-0.17-kolibri-home-with-quizzes.tar.gz…]()

Here's the kolibri home folder I used to test old quizzes in case that helps others in testing

Reply:
Agreed on both accounts @nucleogenesis (re: margin and), yellow color for the notice/alert may be better suited for this case 👍🏽